virttest/libvirt_xml/vm_xml.py

```diff
@@ -2091,6 +2091,18 @@ def remove_numa_cells(self):
             pass  # Element already doesn't exist
         self.xmltreefile.write()
 
+    def remove_elem_by_xpath(self, xpath_to_remove):
+        """
+        Remove a specified sub element from cpu configuration
+
+        :param xpath_to_remove:  str, like '/model' '/numa'
+        """
+        try:
+            self.xmltreefile.remove_by_xpath(xpath_to_remove, remove_all=True)
+        except (AttributeError, TypeError):
+            pass  # Element already doesn't exist
```

Comment:
What if wrong xpath was given to method? There is no indication to higher level method which will call this method. It should be logged at least, I believe.

Reply:
Simply copied from existing codes. But you are right, I updated.

Reply:
The pass statement could be deleted, but it is not harmful, so it can stay there as well.

Reply:
Yes, you are right, I forgot to remove it. Done.